refinery/ui/source/js/commons/partials/download-csv-button.html

```diff
@@ -0,0 +1,17 @@
+<button
+  class="refinery-base btn btn-default btn-xs header-button"
+  ng-click="$ctrl.downloadCsv()">
+  Download CSV <span>({{ $ctrl.totalFiles }} rows)</span>
+</button>
+<span class="p-l-1-3">
+  <a
+    popover-placement="left"
+    uib-popover-html="'Download a .csv file including all files currently
```

Comment:
doesn't need to be html, since it's plain text.

Reply:
"Download all the files in the table below as a .csv. Starting filtering with the...."